spec/cqm/converter/hds_record_spec.rb

```diff
@@ -8,14 +8,13 @@
 
   it 'converts HDS to QDM and maintains id' do
```

Comment:
'converts HDS to CQM and maintains QDM.id' may be slightly more accurate? I'm not totally sure, but that is how it looks to me